src/Storages/System/StorageSystemPartsBase.cpp

```diff
@@ -139,6 +137,14 @@ StoragesInfoStream::StoragesInfoStream(const SelectQueryInfo & query_info, Conte
                         continue;
 
                     String engine_name = storage->getName();
+                    UUID storage_uuid = storage->getStorageID().uuid;
+                    if (database->getEngineName() == "Ordinary")
+                    {
+                        SipHash hash;
+                        hash.update(database_name);
+                        hash.update(table_name);
+                        storage_uuid = hash.get128();
```

Comment:
It should not display any UUIDs for tables from Ordinary databases, otherwise, it may be confusing. Also, assigning a random hash to UUID is wrong, because this UUID is likely to be invalid, see https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)

Reply:
we don't display this UUID - we need it only as a key for internal map - previously we used database_name+table_name pair here, but because we can delete table with the same name multiple times, it can't be used to uniquely identify such tables in dropped_tables, so I replaced this pair with table's UUID

Reply:
I'm confused here, wouldn't we also get the same hash for the same database - table name pair and still have the same problem for Ordinary databases as without UUID?

Reply:
@antonio2368 yes, we would, but per my knowledge Ordinary do not use postponed deletion